GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -663,6 +663,7 @@ private void UpdateVisibleRowRange()
         private async Task UpdateVisibleRowRangeInternalAsync()
         {
             int fromIndex = Math.Max(0, FirstDisplayedScrollingRowIndex);
+            // Rounding up integer division: (a+b-1)/b = ceil(a/b)
             int visibleRowCount = _rowHeight <= 0 ? 0 : (Height + _rowHeight - 1) / _rowHeight;
```

Comment:
It does not compile because a formatting rule enforces an empty line before a comment.
If such a comment is really needed, I do not want that empty line in addition here. As a compromise, the comment could be appended.
```suggestion
            int visibleRowCount = _rowHeight <= 0 ? 0 : (Height + _rowHeight - 1) / _rowHeight; // Rounding up integer division: (a+b-1)/b = ceil(a/b)
```

Reply:
> It does not compile because a formatting rule enforces an empty line before a comment.

Oops, sorry. The risk of editing in the brwoser and not in VS. :-)